visualization/app/codeCharta/ui/dialog/dialog.service.ts

```diff
@@ -45,4 +55,34 @@ export class DialogService {
 
 		return this.$mdDialog.show(prompt)
 	}
+
+	private getWrappedDialog(content: string) {
+		return (
+			'<md-dialog class="import-error-dialog">' +
+			"<md-toolbar>" +
+			'<h2 class="md-toolbar-tools">JSON Schema Error</h2>' +
+			"</md-toolbar>" +
+			'<md-dialog-content class="md-dialog-content">' +
+			content +
+			"</md-dialog-content>" +
+			"<md-dialog-actions>" +
+			'<md-button class="md-primary">Ok</md-button>' +
+			"</md-dialog-actions>" +
+			"</md-dialog>"
+		)
+	}
+
+	private createDialogContent(errors: ErrorObject[]) {
+		let html = ""
+		errors.forEach((error: ErrorObject, index: number) => {
+			const key = _.keys(error.params)[0]
+			const object = " " + JSON.stringify(error.params[key])
+
+			html += "<p><b>Error #" + (index + 1) + "</b> " + key + " <i>" + object + "</i></p>"
```

Comment:
Is the error index useful for anyone? I suggest to remove that.

The error objects seem somewhat uncommon though. How and where are these errors generated?

Reply:
They are generated by the `ajv` json schema validator inside our [fileValidator.ts](https://github.com/MaibornWolff/codecharta/blob/master/visualization/app/codeCharta/util/fileValidator.ts#L30)

Reply:
And yea we can remove the index. Officially we get an array of error objects, in reality I always just got one.